app/components/views/TicketsPage/PurchaseTab/PurchaseTickets/PurchaseForm.jsx

```diff
@@ -4,8 +4,6 @@ import { PassphraseModalButton, KeyBlueButton } from "buttons";
 import { AccountsSelect, NumTicketsInput, VSPSelect } from "inputs";
 import { Balance } from "shared";
 import styles from "../PurchaseTab.module.css";
```

Comment:
Just noting that we should avoid such css modules imports, instead this component should have it's own encapsulated css module.

When we are done with #2439 I'm gonna create a follow up issue listing all components with such problematic import statements.

Reply:
You are right. I noticed it, but I did not want to deal with it in this PR. Although, I strived for creating separate css modules in the legacy ticket purchase part.

Reply:
Yep, thanks for that 👍